app/components/forms/wizard/basic-details-step.js

```diff
@@ -114,13 +114,26 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     this.getForm().form('remove prompt', 'discount_code');
   }),
 
+  donationTickets: computed.filterBy('tickets', 'type', 'donation'),
+
   didInsertElement() {
     if (!this.isCreate && this.get('data.event.copyright') && !this.get('data.event.copyright.content')) {
       this.set('data.event.copyright', this.store.createRecord('event-copyright'));
     }
   },
 
   getValidationRules() {
+    window.$.fn.form.settings.rules.checkMaxMin = () => {
```

Comment:
This should check for a single ticket. Not all tickets.

Reply:
Yes, for a single donation ticket, wouldn't want to check the validations of all other donation tickets